conf/messages-ecc.en

```diff
@@ -22,8 +22,6 @@ eds.existing-eori.enrolment.confirmation.title=Application complete
 eds.existing-eori.enrolment.confirmation.heading=Application complete
 eds.existing-eori.enrolment.confirmation.para1=You are now enrolled to the {0} service.
 
-cds.subscription.outcomes.feedback=What did you think of this service?
```

Comment:
An example of re-using the original content (which includes "(opens in new tab)"